### PR TITLE
[DONE] Ptitloup/feature quizz add user answer

### PR DESCRIPTION
**Pull request by ptitloup**
Before sending your pull request, make sure the following are done:

* [X] You have read our [contribution guidelines](https://github.com/EsupPortail/Esup-Pod/blob/master/CONTRIBUTING.md).
* [X] Your PR targets the `develop` branch.
* [X] The title of your PR starts with `[WIP]` or `[DONE]`.

### CODE REVIEW

**Review by LoicBonavent — APPROVED** [review]
Ok pour moi

File: pod/quiz/static/quiz/js/video-quiz-submit.js [hunk 1]
```diff
@@ -53,9 +53,22 @@ for (let questionElement of questionList) {
...
       }
+      // check if question is "single_choice", "multiple_choice", "short_answer", "long_answer"
       if ((Array.isArray(user_answer) && user_answer.includes(answer.value)) || user_answer === answer.value ){
         answer.checked = true;
       }
     }
   }
+  // case user input
+  let user_input = questionElement.querySelector(`#id_${questionid}-user_answer`);
+  allanswersarr = Array.from(allanswers)
```
Comment (Badatos) [inline pod/quiz/static/quiz/js/video-quiz-submit.js:64]: plutot qu'un querySelector, je ferais plutot un `document.getElementById(id_${questionid}-user_answer)`, vu qu'on a l'id.

**Review by Badatos — APPROVED** [review]
Juste une mini remarque, sinon OK pour moi ;)

File: pod/quiz/static/quiz/js/video-quiz-submit.js (hunk 1 above)
Comment (ptitloup) [reply pod/quiz/static/quiz/js/video-quiz-submit.js:64]: corrigée

**Pull request merged**